concurrency/MpmcQueue.h

```diff
@@ -69,21 +101,19 @@ class MpmcQueue
 public:
     typedef T value_type;
     explicit MpmcQueue(uint32_t capacity)
-        : _entriesPerCacheline((63 + sizeof(Entry)) / sizeof(Entry)),
-          _capacity(_entriesPerCacheline * ((capacity + _entriesPerCacheline - 1) / _entriesPerCacheline)),
-          _blockSize(memory::page::alignedSpace(_capacity * sizeof(Entry))),
-          _elements(reinterpret_cast<Entry*>(memory::page::allocate(_blockSize)))
+        : _capacity(
```

Comment:
Isn't it better to have an assert for capacity==0? What is the point of zero length queues?